src/NServiceBus.Core/Utils/FileVersionRetriever.cs

```diff
@@ -4,19 +4,15 @@
     using System.Diagnostics;
     using System.Reflection;
 
-    /// <summary>
-    /// Helper class to retrieve File version.
-    /// </summary>
     class FileVersionRetriever
     {
-        /// <summary>
-        /// Retrieves a semver compliant version from a <see cref="Type" />.
-        /// </summary>
-        /// <param name="type"><see cref="Type" /> to retrieve version from.</param>
-        /// <returns>SemVer compliant version.</returns>
         public static string GetFileVersion(Type type)
```

Comment:
Minor point, but the whole method could be:

```c#
public static string GetFileVersion(Type type) => GetFileVersion(type.Assembly);
```

Reply:
Ah, good catch, fixed